package.json

```diff
@@ -53,8 +59,9 @@
     "pretest": "npm run qunit-fixture && npm run babel:tests && npm run npmcopy",
     "qunit-fixture": "node build/tasks/qunit-fixture.js",
     "start": "node -e \"require('./build/tasks/build.js').buildDefaultFiles({ watch: true })\"",
+    "test:bundlers": "node -e \"require('./test/bundler_smoke_tests/utils.cjs').cleanTmpBundlersDir()\" && npm run build:all && rollup -c test/bundler_smoke_tests/rollup-pure-esm.config.js && rollup -c test/bundler_smoke_tests/rollup-commonjs.config.js && webpack -c test/bundler_smoke_tests/webpack.config.cjs && node test/bundler_smoke_tests/run-jsdom-tests.js",
```

Comment:
What do you say we clean up tmp afterwards? It probably doesn't need to stick around.

Reply:
I'll look into tweaking the scripts. As for the `tmp` directory itself, it may be sometimes easier to analyze  issues if built files stick around. I can always temporarily remove the removal if I have such a need, though.

I would not remove the folder from ignore files, though. A lot can go wrong during a test and it may not complete, leaving the dirty state. I prefer the state to always be clean.

We can optionally change entries in ignore files/arrays to match all directories named `tmp`, regardless of location - then, we won't need to add more entries if we need another directory.